src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -94,75 +95,95 @@
-                [:span {} (create-inline-form-label "Master Disk Size")]]
+               (flex/box {}
+                [:div {:style {:width "48%" :marginRight "4%" :marginBottom 4}}
+                 [FoundationTooltip {:text (create-inline-form-label "Master Machine Type")
+                                     :tooltip "Determines the number of CPUs and memory for the master node."}]]
+                [:div {:style {:width "48%" :marginBottom 4}}
+                 [FoundationTooltip {:text (create-inline-form-label "Master Disk Size")
+                                     :tooltip "Size of the disk on the master node. Minimum size is 100GB."}]])
                [:div {:display "inline-block"}
                 (style/create-identity-select {:ref "masterMachineType" :style {:width "48%" :marginRight "4%"}
-                                               :defaultValue "n1-standard-4"}
+                                               :defaultValue "n1-standard-4" :data-test-id "master-machine-type-select"}
                   machineTypes)
                 [input/TextField {:ref "masterDiskSize" :autoFocus true :style {:width "41%"}
-                                  :defaultValue 500 :min 0 :type "number"}]
+                                  :defaultValue 500 :min 0 :type "number" :data-test-id "master-disk-size-input"}]
                 [:span {:style {:marginLeft "2%"}} (create-inline-form-label "GB")]]
-               (style/create-form-label "Workers")
+               [:div {:style {:marginBottom 4}} [FoundationTooltip {:text (style/create-form-label "Workers") :tooltip "Workers can be 0, 2 or more. Google Dataproc does not allow 1 worker."}]]
                [input/TextField {:ref "numberOfWorkers" :autoFocus true :style {:width "100%"}
-                                 :defaultValue 0 :min 0 :type "number"}]
-               [:div {:display "inline-block"}
-                [:span {:style {:paddingRight "19%"}} (create-inline-form-label "Worker Machine Type")]
-                [:span {} (create-inline-form-label "Worker Disk Size")]]
+                                 :defaultValue 0 :min 0 :type "number" :data-test-id "workers-input"}]
+               (flex/box {}
+                [:div {:style {:width "48%" :marginRight "4%" :marginBottom 4}}
+                 [FoundationTooltip {:text (style/create-form-label "Worker Local SSDs")
+                                     :tooltip "The number of local solid state disks for workers. Ignored if Workers is 0."}]]
+                [:div {:style {:width "48%" :marginBottom 4}}
+                 [FoundationTooltip {:text (style/create-form-label "Preemptible Workers")
+                                     :tooltip "Ignored if Workers is 0."}]])
+               (flex/box {}
+                         [input/TextField {:ref "numberOfWorkerLocalSSDs" :autoFocus true :style {:width "48%" :marginRight "4%"}
```

Comment:
Check indentation. There's a new version of intellij-clojure-style.xml you can import.